### ISSUE TITLE
Color Contrast Miscalculation

### ISSUE THREAD

**Issue by jkantner** [dequelabs/axe-core#589]
This was originally an issue I submitted to Chrome’s Lighthouse, but they wanted me to present it here instead because Lighthouse’s accessibility tester is powered by aXe.

So the issue is that I’m using `#ccbade` for `<h3>`, `#ffffff` for `<p>`, and `#333333` as a background for `<figcaption>`. I tested these on a color contrast checker such as [Hex Naw](https://hexnaw.com/), and both the `<h3>` and `<p>` colors pass with AAA.
```
<figcaption>
<h3>Heading 3</h3>
<p>Paragraph text</p>
</figcaption>
```
I tested the same with Lighthouse, but it keeps accusing the `<h3>` and `<p>` of not having sufficient contrast with the background. I tried slightly adjusting the text colors, but still nothing changed after retesting.
![screen shot 2017-10-25 at 11 13 59 am](https://user-images.githubusercontent.com/16387543/32006782-a96ca144-b975-11e7-8ea1-67be0354b4aa.png)
I have a feeling there’s something wrong with the contrast calculation. Since I’m not familiar with aXe or how to use it, this is the best I could do to reproduce the issue.

**Comment by marcysutton** [comment]
Hi there, do you have a working demo we can use to test this? I just tried this markup in a local file and aXe didn't flag any of it. I tried it with both inline styles and a <style> block, and there are no violations or review items.

```
<figcaption style="background-color: #333333;">
<h3 style="color: #ccbade;">Heading 3</h3>
<p style="color: #fff;">Paragraph text</p>
</figcaption>
```

**Comment by jkantner** [comment]
Well, you can play with this [CodePen demo](https://codepen.io/jkantner/pen/eeOBxz). I tested those elements with the aXe extension and Lighthouse in a local file myself, and I got no violations. Every time I test this in a project I’m working on though, Lighthouse complains, and sometimes aXe as well but this about the `<h3>`:

> Fix the following:
> Element has insufficient color contrast of 1.8 (foreground color: #ccbade, background color: #ffffff, font size: 12.0pt, font weight: bold)

It thinks the background color is white, and this doesn’t make any sense.

**Comment by marcysutton** [comment]
Which version of Lighthouse are you using, specifically to help us narrow down the axe-core version? I can't reproduce this with the latest axe-core@2.4.2, but it's possible you were using an older version downstream from Lighthouse.

**Comment by jkantner** [comment]
I’m using Lighthouse 2.3.1 in Chrome 62.0.3202.75, so yeah it’s an older version. Then how do I update it?

**Comment by marcysutton** [comment]
I'd say that's a Lighthouse question, since we don't control how consume include axe-core. Thanks for reporting it though!
